src/languages/es.ts

```diff
@@ -785,6 +785,10 @@ export default {
         },
         addBankAccountFailure: 'Ocurrió un error inesperado al intentar añadir la cuenta bancaria. Inténtalo de nuevo.',
     },
+    cardPage: {
+        expensifyCard: 'Tarjeta Expensify',
+        availableSpend: 'Poder de gasto restante',
```

Comment:
Mmmm sounds weird, I would choose: `Dinero restante disponible`, I think is better to say that you still have money to spend

Reply:
waiting for the exact translations from the slack